core/src/modules/configLoaders/index.ts

```diff
@@ -212,8 +227,6 @@ export async function processConfigObjects(
     }
   }
 
-  if (sortError && errors.length === 0) errors.push(sortError.message);
```

Comment:
i think it would be ok to return here if there is any error in sorting and not do the specific parsing. For example, that code also does `throw new Error("The graph is not a DAG");`
We should stop then, too. (though probably with a better error message).

Reply:
I agree. But it leaves the bigger, generic question unanswered: Should we continue _trying_ to validate objects if we weren't able to properly sort them?

Or should we consider properly sorting to be a prerequisite of running through object validation? (i.e. If we find _any_ sorting error, we don't try to load _any_ objects?)

Reply:
The main question would be if we want to support referencing in code config objects that dont exist on code config. If so, then it still might be there and we need to do better messages downstream. If not, then this app is never going to run anyway, so it's probably easier to just throw here.